.github/workflows/build.yml

```diff
@@ -9,24 +9,35 @@ on:
 jobs:
   generate:
     runs-on: ubuntu-latest
-
     strategy:
       matrix:
         node-version: [12.x]
-
     steps:
     - uses: actions/checkout@v1
     - name: Use Node.js ${{ matrix.node-version }}
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - name: npm install, build and lint
-      run: |
-        npm ci
-        npm start
-        npm run lint
+    - run: npm ci
+    - name: Generate types
+      run: npm start
+    - run: npm run lint
       env:
-        CI: true
```

Comment:
Do you remember why we needed this in the first place?...

Reply:
It is defined in the stock Node.js GitHub Action, and I had just left it in here, but I don't _think_ it does anything? Or at least, I couldn't find any mentions to it in npm documentation. Perhaps it's just convention?

Reply:
Yeah, I don't think it's relevant for our builds. We might want to add this in the future though.